packages/styles/forms.css

```diff
@@ -364,7 +363,10 @@ textarea.Field--has-error:focus:hover,
   color: #3c7aae;
 }
 
-.Checkbox__overlay--disabled.Icon--checkbox-checked,
+.Checkbox__overlay--disabled.Icon--checkbox-checked {
+  color: #78a6d8;
```

Comment:
Following our [variable guidelines](https://cauldron.dequelabs.com/style-guidelines#variables) this should be either inheriting from a var or specifying a fallback value. I couldn't find this defined anywhere else so we should probably specify that as fallback value:

```css
color: var(--field-checkbox-disabled-color, #78a6d8)
```